tests/bot/cogs/sync/test_cog.py

```diff
@@ -272,12 +311,15 @@ async def test_sync_cog_on_user_update(self):
 
                     # Don't care if *all* keys are present; only the changed one is required
                     call_args = self.cog.patch_user.call_args
-                    self.assertEqual(call_args[0][0], after_user.id)
-                    self.assertIn("updated_information", call_args[1])
+                    self.assertEqual(call_args.args[0], after_user.id)
+                    self.assertIn("json", call_args.kwargs)
+
+                    self.assertIn("ignore_404", call_args.kwargs)
+                    self.assertTrue(call_args.kwargs["ignore_404"])
```

Comment:
I really like the idea of checking membership first. Makes for a much nicer error should the test fail on it not being there, I should do this too.